### PR TITLE
Issue 143

### PR DESCRIPTION
**Pull request by martukas**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [x] Check for conflict with integration test
- [x] Unit tests pass

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel.

### CODE REVIEW

File: prototype2/efu/HwCheck.h [hunk 1]
```diff
@@ -31,7 +31,7 @@ class HwCheck {
   void setMinimumMTU(int mtu) { minimumMtu = mtu; }
 
   /// Gleaned from MacOS and CentOS and deemed ignore worthy
-  std::vector<std::string> defaultIgnoredInterfaces = {"ppp0", "docker", "ov-"};
+  std::vector<std::string> defaultIgnoredInterfaces = {"ppp0", "docker", "ov-", "wlp4s0"};
```
Comment (mortenjc) [inline prototype2/efu/HwCheck.h]: This is not the intended use. Only those interfaces that we know for sure are irrelevant should be 
included. In stead use the -u 1500 option to the EFU which allows smaller MTUs

File: prototype2/gdgem/configs/vmm3.json [hunk 2]
```diff
@@ -3,23 +3,27 @@
...
   "srs_mappings" :
   [
     {"fecID":1, "vmmID":0, "planeID":0, "strip_offset":0},
     {"fecID":1, "vmmID":1, "planeID":0, "strip_offset":64},
-    {"fecID":1, "vmmID":6, "planeID":0, "strip_offset":128},
-    {"fecID":1, "vmmID":7, "planeID":0, "strip_offset":192},
-    {"fecID":1, "vmmID":10, "planeID":1, "strip_offset":0},
-    {"fecID":1, "vmmID":11, "planeID":1, "strip_offset":64},
+    {"fecID":1, "vmmID":2, "planeID":1, "strip_offset":0},
+    {"fecID":1, "vmmID":3, "planeID":1, "strip_offset":64},
```
Comment (mortenjc) [inline prototype2/gdgem/configs/vmm3.json]: Should this file be edited?

Comment (martukas) [reply prototype2/gdgem/configs/vmm3.json]: I think yes, and this has been modified to conform with doro's latest needs, which... I don't think we have a better criterion. We should probably keep a config file like this along with whatever test data we get, and that can be separate from the "recommended" one here. This should just be a file that is technically valid, as an example.

File: prototype2/efu/HwCheck.h (hunk 1 above)
Comment (martukas) [reply prototype2/efu/HwCheck.h]: I wasn't entirely clear on this -u business and I will first test that it actually fixes the problem for me. Whatever I am doing on my laptop, I am never ever receiving or sending anything via UDP or Kafka on the wifi interface, so why is the EFU adamant about checking up on how that interface is configured? Does it really need to check every interface on the machine?

Comment (mortenjc) [reply prototype2/efu/HwCheck.h]: Yes the intent is to check mtu on all active, non-virtual, interfaces. In a office/laptop environment you never really know if you are using the wifi interface or the wired ethernet.

If the MTU is too small, packets will be fragmented int o 1500 byte packets upon transmission 
and reassembled upon reception. This overhead make performance comparisons unreliable and confusing.

File: prototype2/gdgem/configs/vmm3.json (hunk 2 above)
Comment (mortenjc) [reply prototype2/gdgem/configs/vmm3.json]: But I think Doro changes this file every time she goes to the lab and reconfigures something. So maybe there should be more files with more descriptive names and perhaps dates?

Anyway, are we saving this in the hdf5 files? Might be a good idea.

Comment (martukas) [reply prototype2/gdgem/configs/vmm3.json]: Yes, there is already a ticket for saving this with the h5. I think henceforth Doro will probably modify the one in essdaq more often.

**Pull request merged**